### ISSUE TITLE
Dead Links 

### ISSUE THREAD

**Issue by Zblay** [hak86/android_kernel_huawei_prague#1]
Hello hak86, 

i am very sorry to bother you, but i i've been using your version of Lineage OS for P8 Lite (PRA-LX1) for over six month now (20180218), and saw you made an update (2018 05 10) 

i can't seem to get the download links right (Nor on your site, XDA forums (https://forum.xda-developers.com/p8lite/p8-lite-2017-development/rom-lineageos-14-1-huawei-p8-lite-2017-t3649292/page1)  or GitHub) and i lack the skills to build it myself, this is why i'm contacting you : how can i download the last LOS14.1 image ? maybe upload a .zip version on github so it remains reachable ? 

I'm not complaining, your work is great, thanks a lot, i know this is no hotline, but i've been struggling a little these days, it would be nice if you could help.

Thanks a lot.

**Comment by Zblay** [comment]
After some researchs, i found this, that seems to work ok : https://basketbuild.com/devs/haky86/HuaweiP8Lite2017/ROMS/Android-8.x 

Maybe add the link to your github profile so people (like me) won't bother you ?

**Comment by hak86** [comment]
yes that's my profile at basketbuild. I will re-open my topic today.
for 14.1 I should remake it.
